command/generate_test.go

```diff
@@ -101,3 +102,25 @@ func (s *generateSuite) TestExecuteSuccess() {
 	c := &generateTask{Files: []string{tmpFileBase}}
 	s.NoError(c.Execute(s.ctx, s.comm, s.logger, s.conf))
 }
+
+type Thing struct {
+	Thing string `yaml:"thing"`
+}
+
+type DrSeuss []Thing
+
+func (s *generateSuite) TestMakeJsonOfAllFiles() {
+	thingOne := []byte(`
+{
+  "thing": "one",
+}
+`)
+	thingTwo := []byte(`
+{
+  "thing": "two",
+}
+`)
+	var jsonArray DrSeuss
+	json := makeJsonOfAllFiles([][]byte{thingOne, thingTwo})
+	s.NoError(yaml.Unmarshal(json, &jsonArray))
```

Comment:
is there a test of this with a slice of heterogeneous commands?

Reply:
This just verifies we're hand-twiddling valid json.